testsuite/features/secondary/min_ansible_control_node.feature

```diff
@@ -4,6 +4,9 @@
 @scope_ansible
 Feature: Operate an Ansible control node in a normal minion
 
+  Scenario: Pre-requisite: Deploy test playbooks and inventory file
```

Comment:
I noticed that the full feature gives 3 whitespaces instead of 2 in the steps of the scenario.
Could you fix it, please?

Reply:
Okk :+1: